src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -299,7 +309,7 @@ const ChatInfoScreen = () => {
                   key={index}
                   item={item}
                   onPress={() => onContactPressed(item, params.from)}
-                  fullname={item?.name || item?.username}
+                  fullname={getUsername(item)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in this line is replacing the `fullname` prop value with the result of the `getUsername` function. This change is consistent with the new logic introduced for handling anonymous usernames.
```diff
- fullname={item?.name || item?.username}
+ fullname={getUsername(item)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->